File: C++/Pattern/Pattern12.cpp
#include <iostream>
using namespace std;

int main() 
{
    int nums= 4;
    
    for(int i=1;i<=nums;i++){
      //numbers
      for(int j=1;j<=i;j++){
       cout <<j;
      }
      
      //space
      
      for(int j=1;j<2*(nums-i);j++){
        cout<<" ";
      }
      
      //numbers
      
      for(int j=i;j>=1;j--){
        cout<<j;
      }
      
      cout<<endl;
      
    }
}

File: Javascript/Pattern/Pattern12.js


let n=4;

for(let i=1;i<=n;i++){
  //Number
  let temp="";
  for(let j=1;j<=i;j++){
    temp+=j;
  }
  //space
  for(let j=1;j<2*(n-i);j++){
    temp+=" "
  }
  //Number
  for(let j=i;j>=1;j--){
    temp+=j;
  }
  console.log(temp)
}